rules/react.js

```diff
@@ -0,0 +1,68 @@
+// eslint-plugin-react https://github.com/yannickcr/eslint-plugin-react
+
+module.exports = {
+	"plugins": [ "react" ],
+	"rules": {
+		"react/display-name": [ 2, { "acceptTranspilerName": true } ], // Prevent missing displayName in a React component definition
+		"react/forbid-prop-types": [ 2, { "forbid": [ "any" ] } ], // Forbid certain propTypes
+		"react/jsx-boolean-value": [ 1, "always" ], // Enforce boolean attributes notation in JSX
+		"react/jsx-closing-bracket-location": [ 1, { "location": "after-props" } ], // Validate closing bracket location in JSX
+		"react/jsx-curly-spacing": [ 2, "always" ], // Enforce or disallow spaces inside of curly braces in JSX attributes
+		"react/jsx-handler-names": [ 1, { "eventHandlerPrefix": "handle", "eventHandlerPropPrefix": "on" } ], // Enforce event handler naming conventions in JSX
+		"react/jsx-indent-props": [ 2, "tab" ], // Validate props indentation in JSX
+		"react/jsx-key": [ 2, 2 ], // Validate JSX has key prop when in array or iterator
+		"react/jsx-max-props-per-line": [ 2, { "maximum": 10 } ], // Limit maximum of props on a single line in JSX
+		"react/jsx-no-bind": [ 0 ], // Prevent usage of .bind() and arrow functions in JSX props
+		"react/jsx-no-duplicate-props": [ 2, { "ignoreCase": false } ], // Prevent duplicate props in JSX
+		"react/jsx-no-literals": 1, // Prevent usage of unwrapped JSX strings
```

Comment:
I think we should turn this one off (0), however, I wanted to point it out because when we decided to localize/i18n/etc... this can help us find string literals in our JSX that should be replaced with resource strings.